baselines/jft/input_utils.py

```diff
@@ -25,30 +25,25 @@
 import numpy as np
 import tensorflow as tf
 import tensorflow_datasets as tfds
-import al_utils  # local file import from baselines.jft
 
-SubsetDatasetBuilder = al_utils.SubsetDatasetBuilder
+T_DatasetBuilder = TypeVar("T_DatasetBuilder", bound=deterministic_data.DatasetBuilder)
 
 
 def _get_dataset_builder(
-    dataset: Union[str, tfds.core.DatasetBuilder,
-                   SubsetDatasetBuilder],
+    dataset: Union[str, T_DatasetBuilder],
     data_dir: Optional[str] = None
-) -> Union[tfds.core.DatasetBuilder, SubsetDatasetBuilder]:
+) -> Union[tfds.core.DatasetBuilder, T_DatasetBuilder]:
```

Comment:
This is fine for now, but I think it will be better to have `al_utils.SubsetDatasetBuilder` subclass `tfds.core.DatasetBuilder`. https://github.com/google/uncertainty-baselines/pull/813 is proposing that change, but still WIP. Fixing the current tests is a priority.